docs/tooling.md

```diff
@@ -1,19 +1,19 @@
-# Bevy Quickstart recommended 3rd-party tools
+# Bevy Quickstart Recommended 3rd-Party Tools
```

Comment:
I'm a fan of normal casing for headers / titles. It's what [wikipedia does](https://en.wikipedia.org/wiki/List_of_game_engines), for example.

Not a big deal either way.

Reply:
Can change

Reply:
[`84f29b2` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/84f29b259581be62fa8ebdf8c76c67e2e18a1241)